src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -90878,7 +90878,17 @@ INSERT INTO tab_event_conditions VALUES(5938,'EVENT','ParticleBinDissolvedEventU
 INSERT INTO tab_event_conditions VALUES(5939,'EVENT','ParticleBinDissolvedEventUpperJejunum','Application_PKSim','ParticleBinDissolvedEventConditionUpperJejunum',0);
 INSERT INTO tab_event_conditions VALUES(5940,'EVENT','ParticleBinDissolvedEventUpperJejunum','Application_PKSim','ParticleBinDissolvedEventConditionUpperJejunum',0);
 INSERT INTO tab_event_conditions VALUES(5957,'EVENT','Urinary_Bladder_Emptying_StartEvent','Events','Urinary_Bladder_Emptying_StartEvent_Condition',1);
-CREATE TABLE tab_known_transporters (gene text, species text REFERENCES tab_species, transport_type text, CONSTRAINT tab_known_transporters_dg_tmp_pk PRIMARY KEY (gene, species), CONSTRAINT tab_known_transporters_pk UNIQUE (gene, species));
+CREATE TABLE [tab_active_transport_types]([transport_type] TEXT PRIMARY KEY NOT NULL REFERENCES [tab_process_types]([process_type]) ON UPDATE CASCADE) WITHOUT ROWID;
+INSERT INTO tab_active_transport_types VALUES('BiDirectional');
+INSERT INTO tab_active_transport_types VALUES('Efflux');
+INSERT INTO tab_active_transport_types VALUES('Influx');
+INSERT INTO tab_active_transport_types VALUES('PgpLike');

```

Comment:
Added the new table **tab_active_transport_types** with the 4 currently available active transport types.
So actually it's just a subset of **tab_process_types**